DESTDIR=foo make install does not work

I've been trying to create an AUR package for psmt2-frontend for Arch Linux. However, I've not been able to set `DESTDIR` when invoking `make install`.

Got:

```
[u@x1 psmt2-frontend-0.1]$ export DESTDIR=/home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend
[u@x1 psmt2-frontend-0.1]$ make install
OCAMLFIND_DESTDIR=/usr/lib/ \
  ocamlfind install psmt2-frontend src/psmt2Frontend.* src/*.mli META
ocamlfind: Cannot mkdir /usr/lib/psmt2-frontend: Permission denied
make: *** [Makefile:121: install] Error 2
```

Expected:
```
[u@x1 psmt2-frontend-0.1]$ export DESTDIR=/home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/
[u@x1 psmt2-frontend-0.1]$ make install
OCAMLFIND_DESTDIR=/home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/ \
  ocamlfind install psmt2-frontend src/psmt2Frontend.* src/*.mli META
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/version.mli
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/smtlib_parser.mli
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/options.mli
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/psmt2Frontend.o
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/psmt2Frontend.cmxs
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/psmt2Frontend.cmxa
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/psmt2Frontend.cmx
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/psmt2Frontend.cmt
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/psmt2Frontend.cmo
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/psmt2Frontend.cmi
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/psmt2Frontend.cma
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/psmt2Frontend.a
Installed /home/u/Desktop/ocaml-psmt2-frontend/pkg/psmt2-frontend/lib/psmt2-frontend/META
```

--- a/Makefile.in
+++ b/Makefile.in
@@ -13,7 +13,7 @@
 # (enclosed in the file LGPL).
 
 # where to install the binaries
-DESTDIR=@prefix@
+DESTDIR?=@prefix@
 # prefix=@prefix@
 # exec_prefix=@exec_prefix@
 # BINDIR=$(DESTDIR)@bindir@

--- a/Makefile.in
+++ b/Makefile.in
@@ -17,7 +17,7 @@ DESTDIR?=@prefix@
 # prefix=@prefix@
 # exec_prefix=@exec_prefix@
 # BINDIR=$(DESTDIR)@bindir@
-LIBDIR=$(DESTDIR)/lib/
+LIBDIR?=$(DESTDIR)/lib/
 # DATADIR=$(DESTDIR)@datadir@/psmt2-frontend
 
 # where to install the man page

